dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/Strain.java

```diff
@@ -125,32 +133,33 @@ private Gene loadAnnotations(Annotation annotation, Gene prev) {
 				(x, acc) -> x <= acc);
 		ClusterDrawable right = getClusterDrawable(annotation, Double.MIN_VALUE,
 				(x, acc) -> x >= acc);
-		if (left != null) {
-			childContent.getChildren().add(new Connection(g.translateXProperty().add(0),
+		/*if (left != null) {
+			content.getChildren().add(new Connection(g.translateXProperty().add(0),
 					g.translateYProperty().add(0), left.translateXProperty().add(0),
 					left.translateYProperty().add(0)));
 		}
 		if (right != null) {
-			childContent.getChildren().add(new Connection(g.translateXProperty().add(
+			content.getChildren().add(new Connection(g.translateXProperty().add(
 					g.widthProperty()), g.translateYProperty().add(0),
 					right.translateXProperty().add(0), right.translateYProperty().add(0)));
-		}
+		}*/
```

Comment:
Will be taken care of in redo-annotations. It's commented because I cherry-picked 027df20 from that branch. I can fix it, if you want?